unicodetools/src/main/java/org/unicode/props/UcdPropertyValues.java

```diff
@@ -1831,6 +1831,7 @@ public enum Script_Values implements Named {
         Math_Symbols("Zmth"),
         Emoji_Symbols("Zsye"),
         Other_Symbols("Zsym"),
+        Sidetic("Qabw"),
```

Comment:
This file should ideally be generated from PVA.txt by GenerateEnums.java. (And then you need to reformat using `mvn spotless apply` ...)

https://github.com/unicode-org/unicodetools/blob/main/docs/newunicodeproperties.md#run-generateenumsjava

Reply:
I was following https://github.com/unicode-org/unicodetools/blob/main/docs/build.md#new-blocks which starts from the Java rather than PVA; I guess that section is outdated?

Reply:
Maybe I am just outdated. I certainly don't remember all of the docs that we have...
Looking at that:

> Add to ShortBlockNames.txt (you need to know what the short name is, you can find it in PropertyValueAliases.txt)

This suggests that PVA.txt wants to be updated first.

My comment about how it *should* work is a wish for the future. Clearly you updated the seemingly-redundant internal file, and the tools added the block to PVA.txt. But what I *wish* for, in some future, is to clearly have PVA.txt be an input, and not to need SBN.txt.

> Add long & short names to UcdPropertyValues.java enum Block_Values

Again, clearly you can do this manually, but if you do it differently from GenerateEnums+spotless, then you should get a CI check failure.

Reply:
> Again, clearly you can do this manually, but if you do it differently from GenerateEnums+spotless, then you should get a CI check failure.

Generally it sounds like we need a CI check for GenerateEnums, as you noted on #416. We can discuss that tomorrow night.

(FWIW spotless fails for me with an issue that looks like `https://github.com/google/google-java-format/issues/787`)

Reply:
> > Again, clearly you can do this manually, but if you do it differently from GenerateEnums+spotless, then you should get a CI check failure.
> 
> Generally it sounds like we need a CI check for GenerateEnums, as you noted on #416. We can discuss that tomorrow night.

We have one: https://github.com/unicode-org/unicodetools/blob/main/.github/workflows/cli-build-instructions.yml#L203

Reply:
Ah right, it just runs after the UCA validation, which fails.